etc/DependencyInstaller.sh

```diff
@@ -71,19 +89,28 @@ _installUbuntuPackages() {
     export DEBIAN_FRONTEND="noninteractive"
     apt-get -y update
     apt-get -y install --no-install-recommends \
+        bison \
         curl \
+        flex \
+        help2man \
+        libfl-dev \
+        libfl2 \
+        libgoogle-perftools-dev \
```

Comment:
Does Verilator require this?

Reply:
It's optional, but "should be installed for good performance" ([Verilator install prerequisites](https://verilator.org/guide/latest/install.html#install-prerequisites))